docs/source/settings.rst

```diff
@@ -359,6 +367,11 @@ speed up server boot times. Although, if you defer application loading
 to each worker process, you can reload your application code easily by
 restarting workers.
 
+.. note::
```

Comment:
i'm not sure we should discourage people to se async with --preload but rather warn than any monkey patching will be done once in the worker. So afaik what is discouraged is to monkey-patch globally inside a module. Thoughts?